src/components/NavBar.js

```diff
@@ -0,0 +1,9 @@
+export default function NavBar() {
+  return (
+  <ul className="navbar">
+    <li><a style={{textDecoration: 'none'}} href="/">Home |</a></li>
+    <li><a style={{textDecoration: 'none'}} href="/calculator">Calculator |</a></li>
+    <li><a style={{textDecoration: 'none'}} href="/quotes">Quotes</a></li>
+  </ul>
```

Comment:
- [ ] It would be nice if your app behaves like a Single Page Application means it should not reload when we click on any link in the navigation bar to go to different parts of your app :thinking: because now it reloads the page when i click on any of it :fearful: Kindly fix it by using `Navlink` instead of `a` tag which comes from React Router :+1: I have used it below :arrow_down:

```javascript
import { NavLink } from "react-router-dom"

export default function NavBar() {
  return (
  <ul className="navbar">
    <li><NavLink style={{textDecoration: 'none'}} to="/">Home |</NavLink></li>
    <li><NavLink style={{textDecoration: 'none'}} to="/Calculator">Calculator |</NavLink></li>
    <li><NavLink style={{textDecoration: 'none'}} to="/Quotes">Quotes</NavLink></li>
  </ul>
  )
}
```